src/sidebars/favourites/tearout.js

```diff
@@ -0,0 +1,218 @@
+(function(window) {
+    'use strict';
+
+    angular.module('openfin.tearout', ['openfin.geometry', 'openfin.hover', 'openfin.store'])
+        .directive('tearable', ['geometryService', 'hoverService', 'storeService', function(geometryService, hoverService, storeService) {
+            return {
+                restrict: 'C',
+                link: function(scope, element, attrs) {
+                    var tileWidth = 230,
+                        tileHeight = 100;
+
```

Comment:
There might be a way to get this information from the element by traversing the DOM.. not sure though I'll have a look.